Sources/News.php

```diff
@@ -225,11 +225,11 @@ function ShowXmlFeed()
 	// Allow mods to add extra namespaces and tags to the feed/channel
 	$namespaces = array(
 		'rss' => array(),
-		'rss2' => array('atom' => 'http://www.w3.org/2005/Atom'),
-		'atom' => array('' => 'http://www.w3.org/2005/Atom'),
+		'rss2' => array('atom' => 'https://www.w3.org/2005/Atom'),
+		'atom' => array('' => 'https://www.w3.org/2005/Atom'),
```

Comment:
Good enough for me.  I already reversed it.  Just didn't know if any official documentation existed supporting it either way.